juju/juju-restore
# Added confirm message and related tests.

## Description of change

Confirm message now provides details from a backup file (still to be filled with the actual data from the backup file when it's unpacked and read).

Added tests for command and its inputs/outputs.

## QA steps

1. Run command. At confirmation, check messages and abort. 
2. Run command. At confirmation, check messages and choose to proceed.
